sass/themes/schemas/components/light/_combo.scss

```
@@ -19,15 +19,22 @@
 /// @prop {Map} toggle-button-foreground [color: ('gray', 700)] - The combo toggle button foreground color.
 /// @prop {Map} toggle-button-foreground-focus [color: ('gray', 700)] - The combo toggle button foreground color when the combo is focused.
 /// @prop {Map} toggle-button-foreground-filled [color: ('gray', 900)] - The combo toggle button foreground color when the combo is focused.
-/// @prop {Map} toggle-button-background-disabled [color: ('gray', 300)] - The combo toggle button background color when the combo is disabled.
+/// @prop {Map} toggle-button-background-disabled [color: ('gray', 100)] - The combo toggle button background color when the combo is disabled.
 /// @prop {Map} toggle-button-foreground-disabled [color: ('gray', 700)] - The combo toggle button foreground color when the combo is disabled.
+/// @prop {Map} toggle-button-background-focus--border [color: ('gray', 300)] The combo toggle button background color when the combo is focused in material border variant.
```

This is getting way too long, why do we have background and border in the same name? How is the border related to the background in any way?

Can this just be `toggle-button-border-focus`?

```suggestion
/// @prop {Map} toggle-button-border-focus [color: ('gray', 300)] - The combo toggle button background color when the combo is focused in material border variant.
```

---

No, this is related to the material input of type border only, see the description